test/core/tableTests.ts

```diff
@@ -202,24 +202,25 @@ describe("Tables", () => {
     assert.isFalse(table.fixedHeight(), "height unfixed now that a subcomponent has unfixed height");
   });
 
-  it.skip("table.requestedSpace works properly", () => {
-    // [0 1]
-    // [2 3]
+  it("table.requestedSpace works properly", () => {
     var c0 = new Plottable.Component();
     var c1 = TestMethods.makeFixedSizeComponent(50, 50);
     var c2 = TestMethods.makeFixedSizeComponent(20, 50);
     var c3 = TestMethods.makeFixedSizeComponent(20, 20);
 
-    var table = new Plottable.Components.Table([[c0, c1], [c2, c3]]);
+    var table = new Plottable.Components.Table([
+      [c0, c1],
+      [c2, c3]
+    ]);
 
     var spaceRequest = table.requestedSpace(30, 30);
-    TestMethods.verifySpaceRequest(spaceRequest, 30, 30, "1");
+    TestMethods.verifySpaceRequest(spaceRequest, 70, 100, "1");
 
     spaceRequest = table.requestedSpace(50, 50);
-    TestMethods.verifySpaceRequest(spaceRequest, 50, 50, "2");
+    TestMethods.verifySpaceRequest(spaceRequest, 70, 100, "2");
 
     spaceRequest = table.requestedSpace(90, 90);
-    TestMethods.verifySpaceRequest(spaceRequest, 70, 90, "3");
+    TestMethods.verifySpaceRequest(spaceRequest, 70, 100, "3");
```

Comment:
Yes, the method Table.requestedSpace returns something else now (or it always did). It returns the minimum required space. The `Table` will always need 70 x 100 pixels, because of the fixed components. I believe it will make more sense when we approach https://github.com/palantir/plottable/issues/2059